tests/RPC/RPCMessage.test.ts

```diff
@@ -0,0 +1,23 @@
+import PeerInfo from '@polykey/PeerStore/PeerInfo'
+import RPCMessage from '@polykey/RPC/RPCMessage'
+
+// TODO add tests as part of testing PR
```

Comment:
More tests needed!! But after we get the domain modelling correct.

Reply:
yeah definitely, I think the amount of testing required for this library requires a separate PR? I  can add it after these two are merged, I will try to add more tests now too :+1: